.github/workflows/build-and-test.yml

```diff
@@ -150,12 +150,11 @@ jobs:
           trap "go-junit-report -set-exit-code < test-results/junit/unittests.out > test-results/junit/unittests.xml" EXIT
           # some hostmetrics scraping tests require access to /proc.
           sudo make gotest | tee test-results/junit/unittests.out
-      - name: Upload Unit Test Results
+      - name: Publish Unit Test Results
         if: always()
-        uses: actions/upload-artifact@v1
+        uses: EnricoMi/publish-unit-test-result-action@v1.9
         with:
-          name: unittests-results
-          path: test-results/junit/unittests.xml
+          files: test-results/junit/unittests.xml
```

Comment:
What does this do? Is it supposed to publish the results somewhere visible?

Reply:
Trying to figure that out, for the moment there is a permission issue that I hope I fixed